### PR TITLE
Bump up the stack to 7.13

### PR DESCRIPTION
**Pull request by mtojek**
## What does this PR do?

This PR updates the dependency on elastic-package to use the 7.13.0-SNAPSHOT stack.

## Checklist

~- [ ] I have reviewed [tips for building integrations]~(https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
~- [ ] I have verified that all data streams collect metrics or logs.~
~- [ ] I have added an entry to my package's `changelog.yml` file.~

## How to test this PR locally

CI will verify it.

### CODE REVIEW

**Comment by mtojek** [comment]
It looks like I discovered more fields in 7.13.0-SNAPSHOT for the Kubernetes integration: https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations/detail/PR-745/1/tests

File: packages/kubernetes/data_stream/state_node/fields/fields.yml [hunk 1]
```diff
@@ -12,6 +12,15 @@
           type: boolean
           description: |
             Node unschedulable status
+        - name: disk_pressure
+          type: boolean
```
Comment (ChrsMark) [inline packages/kubernetes/data_stream/state_node/fields/fields.yml:16]: Looking into this I found we have an issue with the mapping. Opened a PR to fix it: https://github.com/elastic/beats/pull/24241. Would you mind to adjust this one accordingly?

Comment (mtojek) [reply packages/kubernetes/data_stream/state_node/fields/fields.yml:16]: I can't do it right now due to type check (expecting boolean). We can do it later, unfortunately, once master starts failing. Keep in mind that we'll have to wait for the Beats and Agent's image to be built first.

Comment (ChrsMark) [reply packages/kubernetes/data_stream/state_node/fields/fields.yml:16]: Ah right!

Comment (mtojek) [reply packages/kubernetes/data_stream/state_node/fields/fields.yml:16]: We discussed this issue privately, seems that it should be keyword (values: true, false, _unknown_). I will try to adjust the PR.

File: packages/kubernetes/data_stream/state_node/fields/fields.yml [hunk 2]
```diff
@@ -12,6 +12,15 @@
           type: boolean
           description: |
             Node unschedulable status
+        - name: disk_pressure
+          type: keyword
+          description: Node DiskPressure status
```
Comment (ChrsMark) [inline packages/kubernetes/data_stream/state_node/fields/fields.yml:17]: Can we add possible values in the description plz like at https://github.com/elastic/beats/pull/24241/files#diff-d1f0b4526ef0b5e6ec951c4fcd427a0aa9caa0f6175e430a7d06f6b237dc6d81R30239?
```suggestion
          description: Node DiskPressure status (true, false or unknown)
```

File: packages/kubernetes/data_stream/state_node/fields/fields.yml [hunk 3]
```diff
@@ -12,6 +12,15 @@
...
+        - name: memory_pressure
+          type: keyword
+          description: Node MemoryPressure status
```
Comment (ChrsMark) [inline packages/kubernetes/data_stream/state_node/fields/fields.yml:20]:
```suggestion
          description: Node MemoryPressure status (true, false or unknown)
```

File: packages/kubernetes/data_stream/state_node/fields/fields.yml [hunk 4]
```diff
@@ -12,6 +12,15 @@
...
+        - name: pid_pressure
+          type: keyword
+          description: Node PIDPressure status
```
Comment (ChrsMark) [inline packages/kubernetes/data_stream/state_node/fields/fields.yml:23]:
```suggestion
          description: Node PIDPressure status (true, false or unknown)
```

**Review by ChrsMark — APPROVED** [review]
lgtm, only one nit. Thanks!

**Review by ChrsMark — APPROVED** [review]
:shipit:

**Pull request merged**